### PR TITLE
Add windows configuration for sharded tests.

### PR DESCRIPTION
**Pull request by godofredoc**
Bugs:
  https://github.com/flutter/flutter/issues/63258
  https://github.com/flutter/flutter/issues/63242
  https://github.com/flutter/flutter/issues/63241

### CODE REVIEW

File: config/framework_config.star [hunk 1]
```diff
@@ -218,21 +212,118 @@ def framework_prod_config(platform_args, branch, version, ref):
...
+            swarming.cache(name = "pub_cache", path = ".pub_cache"),
+            swarming.cache(name = "android_sdk", path = "android29"),
+        ],
+    )
+    common.windows_prod_builder(
+        name = "Windows%s SDK Drone|frwdrn" % ("" if branch == "master" else " " + branch),
+        recipe = drone_recipe_name,
+        console_view_name = None,
+        no_notify = True,
+        priority = 30 if branch == "master" else 25,
```
Comment (keyonghan) [inline config/framework_config.star:312]: shall this be added to other builders as well?

File: config/framework_config.star [hunk 2]
```diff
@@ -109,14 +109,8 @@ def framework_prod_config(platform_args, branch, version, ref):
...
-    common.linux_prod_builder(
-        name = "Linux%s|frwk" % ("" if branch == "master" else " " + branch),
-        recipe = recipe_name,
-        console_view_name = console_view_name,
-        triggered_by = [trigger_name],
-        triggering_policy = triggering_policy,
-        **platform_args["linux"]
-    )
+
+    # Linux platform
```
Comment (christopherfujino) [inline config/framework_config.star:113]: What happened to Linux?

File: config/framework_config.star (hunk 1 above)
Comment (godofredoc) [reply config/framework_config.star]: Actually it needs to be removed.

File: config/framework_config.star (hunk 2 above)
Comment (godofredoc) [reply config/framework_config.star:113]: Now it is called Linux framework_tests and running with the new recipes.

**Pull request merged**